src/features/core/cookieConsent/OneTrust/browser.js

```diff
@@ -6,9 +6,10 @@ class OneTrust {
     // If user does not load onetrust sdk before loading rudderstack sdk
     // we will not be filtering any of the destinations.
     if (!window.OneTrust || !window.OnetrustActiveGroups) {
-      throw new Error(
+      logger.error(
```

Comment:
This should be a warning message.

Reply:
But this assumes the user has loaded this integration as they were expecting OneTrust to work, feels like more of error to me

Reply:
Makes sense. @MoumitaM, please revert the change.

Reply:
Addressed